pages/common/terraform-plan.md

```diff
@@ -0,0 +1,32 @@
+# terraform plan
+
+> Generate and show an execution plan.
+> More information: <https://www.terraform.io/docs/cli/commands/plan.html>.
+
+- Generate and show an execution plan:
+
+`terraform plan`
+
+- Destroy mode - Show plan to destroy all remote objects that currently exist:
+
+`terraform plan -destroy`
+
+- Refresh-only mode - Show plan to update the Terraform state and output values:
+
+`terraform plan -refresh-only`
+
+- Specify values for input variables:
+
+`terraform plan -var 'name=value'`
```

Comment:
The `{{}}` will hightlight arguments. Also, can you specify mutliple pairs like this?
```suggestion
`terraform plan -var '{{name1}}={{value1}}' '{{name2}}={{value2}}'`
```

Reply:
Thank you for reviewing.
`-var` accepts only one argument. If you want to pass several arguments, do like this.

```bash
terraform plan -var '{{name1=value1}}' -var '{{name2=value2}}'
```

I'll fix it as above.